Resources/Textures/Clothing/Uniforms/procedural.rsi/meta.json

```diff
@@ -0,0 +1,159 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Created by TheShuEd for Space Station 14",
+  "size": {
```

Comment:
Are these derived from anything or entirely original? They look similar to existing sprites.

Reply:
I drew everything myself anyway, albeit based on the current sprites. I broke it down into layers, masks and such. Every sprite is 50% or more edited by me in one way or another

Reply:
Should still credit the original ones if you used them as a base.

Reply:
Done